### ISSUE TITLE
AWS Transcriptions microservice

### ISSUE THREAD

**Issue by seth-shaw-asu** [asulibraries/islandora-repo#599]

**Comment by seth-shaw-asu** [comment]
I can't seem to get the credentials file into the container, so I need to update https://github.com/asulibraries/AwsTranscribe to read the credentials into the TranscribeServiceClient another way. The constructor can take the credentials, so I'm looking into other options for setting the application configuration, such as [Symfony's secrets](https://symfony.com/doc/current/configuration/secrets.html).

**Comment by seth-shaw-asu** [comment]
Or perhaps better yet, simply have the Dockerfile add the right [environment variables](https://docs.aws.amazon.com/sdk-for-php/v3/developer-guide/guide_credentials_environment.html).

**Comment by seth-shaw-asu** [comment]
I was able to get the credentials file into the container after all, just not how I originally expected to. I also did a bit of code clean-up [an isle branch](https://github.com/asulibraries/AwsTranscribe/tree/isle).

Unfortunately, this microservice works based on the assumption that the files will live in a Fedora 5 (modeshape binary) structured s3 bucket, something that we don't do for the local dev docker build. As this microservice isn't critical for most local-development priorities for the time being and the code will eventually need to be updated for Fedora 6 anyway, it would be prudent to _delay further work on this until we have Fedora 6 working in Docker in staging_.
